packages/jest-emotion/test/printer.test.js

```diff
@@ -0,0 +1,91 @@
+import React from 'react'
+import renderer from 'react-test-renderer'
+import prettyFormat from 'pretty-format'
+import * as emotion from 'emotion'
+import { createSerializer } from '../src'
+
+const { ReactElement, ReactTestComponent, DOMElement } = prettyFormat.plugins
+
+describe('jest-emotion with dom elements', () => {
+  const emotionPlugin = createSerializer(emotion)
+
+  const divStyle = emotion.css(`
```

Comment:
Could you change this to use `css` as a tagged template literal like this?
```jsx
css`
  color: red;
`
```

Reply:
fixed